morib/src/main/java/org/morib/server/domain/allowedSite/application/FetchTabNameServiceImpl.java

```diff
@@ -16,7 +18,7 @@ public String fetch(String url) {
             return doc.title();
         }
         catch (IOException e) {
-            throw new IllegalArgumentException("유효하지 않은 url입니다.");
+            throw new InvalidURLException(ErrorMessage.INVALID_URL);
```

Comment:
이 부분 관련하여서 Jsoup.connct(url).get() 에 대하여 exception이 터질 때! InvalidURLException도 있겠지만! 이외의 케이스에 대한 예외도 발생할 수 있다고 봅니다! 

URL은 맞았지만 사실 네트워크 상의 오류가 발생하여서 일어날 수 있는 경우도 있다고 해서 조금 포괄적으로 묶어주는 예외를 던져줘야 할 거 같아요!

ex) 요청 상 오류가 발생했다? 

이런 의미를 내포하고 있는 친구가 좋아보이는데요! 영재님께서는 어떻게 생각하시나요?

Reply:
좋은 시선 감사합니다! 그럼 catch문을 하나 더 둬서 500 에러로 던지는 코드를 작성해야될 것 같아요!
```suggestion
catch (IOException e) {
            throw new InvalidURLException(ErrorMessage.INVALID_URL);
        }
        catch (Exception e) {
            throw new IllegalStateException("요청에 오류가 발생했습니다.");
        }
```
요렇게 수정해보면 어떨까요?

Reply:
네네! 커스텀으로 해주면 좋겠지만 이런식으로 포괄적으로 처리해주는 방식도 저는 괜찮다고 생각합니다!

Reply:
머지하셔도 될 거 같아요!